client/user.go

```diff
@@ -117,3 +154,103 @@ func (h *V1Client) CreateUser(user *models.V1UserEntity) (string, error) {
 	}
 	return *resp.Payload.UID, nil
 }
+
+// UpdateUser update existing user.
+func (h *V1Client) UpdateUser(uid string, user *models.V1UserUpdateEntity) error {
+	param := clientv1.NewV1UsersUIDUpdateParams().WithUID(uid).WithBody(user)
+	_, err := h.Client.V1UsersUIDUpdate(param)
```

Comment:
just return h.Client.V1UsersUIDUpdate(param)

Reply:
h.Client.V1UsersUIDUpdate(param) will return 2 arguments (*V1UsersUIDUpdateNoContent, error) hence we are returning only err

Reply:
right, ok. the error check is still not needed. You can just return err.

Reply:
done